data/json/recipes/weapon/bashing.json

```diff
@@ -189,6 +189,33 @@
     "tools": [ [ [ "drift", -1 ] ] ],
     "components": [ [ [ "stick_long", 1 ] ], [ [ "fur", 2 ], [ "leather", 2 ] ] ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "survivor_polehammer",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_BASHING",
+    "skill_used": "fabrication",
+    "difficulty": 1,
+    "time": "30 m",
+    "autolearn": true,
+    "qualities": [ { "id": "CUT", "level": 2 }, { "id": "HAMMER", "level": 1 } ],
+    "proficiencies": [ { "proficiency": "prof_carving", "time_multiplier": 1.5, "skill_penalty": 0.15 } ],
+    "components": [ [ [ "spear_shaft", 1 ] ], [ [ "hammer", 1 ], [ "survivor_polehammer_makeshift", 1 ] ], [ [ "nails", 3, "LIST" ] ] ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "survivor_polehammer_makeshift",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_BASHING",
+    "skill_used": "fabrication",
+    "time": "15 m",
+    "autolearn": true,
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "proficiencies": [ { "proficiency": "prof_carving", "time_multiplier": 1.5, "skill_penalty": 0.15 } ],
+    "components": [ [ [ "stick_long", 1 ], [ "long_pole", 1 ] ], [ [ "hammer", 1 ] ] ]
+  },
```

Comment:
How does this recipe work? What are you doing? Is it removing the head from the hammer, carving an appropriate place to mount it, and sticking the head on? How are you doing that in 15-20 minutes?

Reply:
Exactly what you described. Removing hammerhead can be done in 2 minutes. For carving place to mount it, you only need to carve like 3 cm lenght, and not much in diameter

Reply:
How are you removing a hammer head in two minutes with (at worst) a sharp rock? I'd expect at least hammering required for that.

Reply:
Fair enough, I will adjust it when I get in front of computer again